DEVELOPMENT.md

```diff
@@ -30,61 +43,60 @@ flowchart LR
 ```
 
 During test run, the test suites:
-1. create ephemeral pipelines and queues for a given [Buildkite Agent Cluster](https://buildkite.com/docs/clusters/overview).
-2. Run executor, which will monitor jobs from the target queue in target Buildkite Cluster,
-   starts new Jobs in a Kubernetes cluster.
-3. Test suite will clean up those ephemeral objects in the end.
+1. Create ephemeral pipelines and queues for a given [Buildkite Agent Cluster](https://buildkite.com/docs/clusters/overview).
+2. Run executor, which will monitor jobs from the target queue in target Buildkite Cluster and start new Jobs in a Kubernetes cluster.
+3. Clean up those ephemeral objects in the end.
 
-To run integration test locally, we recommend you to run individual test. For example,
+To run integration test locally, we recommend you to run individual tests. For example,
 
 ```bash
 just test -run TestWalkingSkeleton
 ```
 
-## Requirements
+### Integration test requirements
 
 In addition to the usual requirements, the integration tests make use of a GraphQL-enabled Buildkite API token in order to create pipelines, start builds, read logs, and clean up afterwards.
 
-## Setup
+### Setup
+
+Any Buildkite user who has an access to a Kubernetes cluster should be able to run our integration test.
 
-Any member of the public should be able to run our integration as long as you are an user of Buildkite, and you have
-access to a Kubernetes cluster.
+To get the integration test running locally, you will need:
 
-Concretely, to get the integration test running locally, you will need:
-1. A valid Buildkite API token (presuming you are a customer of Buildkite).
-2. A valid Buildkite Agent Token in your target Buildkite Cluster. The agent token needs to be installed in your K8s
+1. A valid Buildkite API token.
+2. A valid Buildkite Agent Token in your target Buildkite Cluster. The agent token needs to be installed in your Kubernetes
```

Comment:
(Comment about the existing document) Since the instructions for installing the token are further down, I don't think this sentence ("The agent token needs to be installed ...") adds value.

Reply:
Done, thanks, @DrJosh9000!